src/test/spec/unified_runner/operation.rs

```diff
@@ -246,7 +246,7 @@ impl Deref for Operation {
 pub(super) struct DeleteMany {
     filter: Document,
     #[serde(flatten)]
-    options: Option<DeleteOptions>,
+    options: DeleteOptions,
```

Comment:
The combination of `#[serde(flatten)]` and wrapping these fields in an `Option` was silencing certain deserialization errors.